internal/backend/rclone/backend_test.go

```diff
@@ -0,0 +1,90 @@
+package rclone_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/restic/restic/internal/backend/rclone"
+	"github.com/restic/restic/internal/backend/test"
+	"github.com/restic/restic/internal/errors"
+	"github.com/restic/restic/internal/restic"
+	rtest "github.com/restic/restic/internal/test"
+)
+
+const rcloneConfig = `
+[local]
+type = local
+`
+
+func newTestSuite(t testing.TB) *test.Suite {
+	dir, cleanup := rtest.TempDir(t)
+
+	return &test.Suite{
+		// NewConfig returns a config for a new temporary backend that will be used in tests.
+		NewConfig: func() (interface{}, error) {
+			cfgfile := filepath.Join(dir, "rclone.conf")
+			t.Logf("write rclone config to %v", cfgfile)
+			err := ioutil.WriteFile(cfgfile, []byte(rcloneConfig), 0644)
+			if err != nil {
+				return nil, err
+			}
+
+			t.Logf("use backend at %v", dir)
+
+			repodir := filepath.Join(dir, "repo")
+			err = os.Mkdir(repodir, 0755)
+			if err != nil {
+				return nil, err
+			}
+
+			cfg := rclone.NewConfig()
+			cfg.Program = fmt.Sprintf("rclone --config %q", cfgfile)
```

Comment:
You can remove `--config %q` and the saving of the file above and just use

`cfg.Remote = repodir`

rclone remotes are either remote:remote/path or /local/path.  (There is a little bit of ambiguity there I'm sure you'll notice which you can workaround with ./localdirwithcolon:/path)

Reply:
Ah, nice, I didn't know that.